### PR TITLE
Reducing the end npm bundle size by switching to use the single lodas…

### PR DESCRIPTION
**Pull request by boxman0617**
…h modules

that are being used instead of the entire lodash library.

screen shot 2018-02-02 at 2 53 21 pm
^ that is the entire `lodash` lib being included within this lib, by allowing this PR, we can reduce that
by quite a lot as you only use 5 functions from lodash.

On another note, would love to become a core contributor to this library! I think you have created an awesome package and want to help maintain it and make it better. Let me know what you think!

### CODE REVIEW

**Review by chasedut — APPROVED** [review]
LGTM !

**Review by hacknightly — APPROVED** [review]

**Review by dw2 — APPROVED** [review]
Looks awesome to me. Nice catch on "userId" naming as well.

File: src/index.js [hunk 1]
```diff
@@ -4,7 +4,11 @@
  * @package react-amplitude
  * @author  Rory Garand <rory@mettleup.com>
  */
-import {isFunction, isNil, isPlainObject, isString, isNull} from 'lodash';
+import isFunction from 'lodash.isfunction';
+import isNil from 'lodash.isnil';
+import isPlainObject from 'lodash.isplainobject';
+import isString from 'lodash.isstring';
+import isNull from 'lodash.isnull';
```
Comment (timurista) [inline src/index.js:11]: this should help with bundle size! LGTM

**Review by timurista — APPROVED** [review]

**Comment by shawnphoffman** [comment]
Bump. It would be really nice to get these changes merged in and pushed to the NPM package.

**Comment by rorygarand** [comment]
Yikes! I forgot about this... Just a sec.

**Review by rorygarand — APPROVED** [review]

**Comment by rorygarand** [comment]
I'll update `npm` shortly.

**Pull request merged**